crates/polars-plan/src/dsl/function_expr/datetime.rs

```diff
@@ -64,6 +64,11 @@ pub enum TemporalFunction {
         closed: ClosedWindow,
     },
     Combine(TimeUnit),
+    #[cfg(feature = "timezones")]
+    DatetimeFunction {
```

Comment:
Not the ideal name here - but `Datetime` is taken by the `datetime` _method_. We should probably do a refactor to separate the methods and functions, but let's save that for another PR.